### PR TITLE
updates to look of app + state variables added

### PR DESCRIPTION
**Pull request by b-bold**
## What I've added: 
- some styling to the login and register pages
- created a 'hello world' page
- login & register pages have naive validations on all input fields
- login page's 'login' button does not work if no inputs fields have values or when some are missing
- register page's 'login' button does not work if some values are not provided 

## Known Bug
- register page 'login' button shows the 'hello world' page when all input fields have no values (let me know if you don't think this is a bug)

## Preview:
Screen Shot 2023-08-05 at 9 01 55 PM

errors on login page: 
Screen Shot 2023-08-05 at 9 02 22 PM

errors on register page: 
Screen Shot 2023-08-05 at 9 03 47 PM

'hello world' welcome page: 
Screen Shot 2023-08-05 at 9 04 24 PM

### CODE REVIEW

File: src/Login.jsx [hunk 1]
```diff
@@ -1,23 +1,35 @@
...
+                <div className="auth-form-container">
+                    <h2> Login </h2>
+                    <form className="login-form" onSubmit={handleSubmit}>
+                        <label htmlFor="email">Email </label>
+                        <input value={email} type="email" placeholder="youremail@gmail.com" name="email"/>
+                        <label htmlFor="password">Password </label>
+                        <input value={pass} type="password" placeholder="*****" name="password" />
+                        <button className='login-btn'>Log In</button>
+                    </form>   
+                    <button className='register-btn' onClick={() => props.onFormSwitch('register')}>Register Here </button>
```
Comment (RealWeeks) [inline src/Login.jsx]: break this handler out into it's own method

File: src/Register.jsx [hunk 2]
```diff
@@ -1,5 +1,37 @@
...
+                        <input value={email} type="email" placeholder="youremail@gmail.com" name="email" />
+                        <label htmlFor="password">Password </label>
+                        <input value={pass} type="password" placeholder="*****" name="password" />
+                        <button className='login-btn'>Log In</button>
+                    </form>
+                    <button className='register-btn' onClick={() => props.onFormSwitch('login')}>Login Here </button>
+                </div>
+            }
+
+            {page && <Welcome />}
```
Comment (RealWeeks) [inline src/Register.jsx]: make this an if/else, as oppose to the two if's it is now

File: src/Register.jsx [hunk 3]
```diff
@@ -1,5 +1,37 @@
...
+                    <form className="register-form" onSubmit={handleSubmit}>
+                        <label>Full Name </label>
+                        <input value={name} name="name" id="id" placeholder="full name" />
+                        <label htmlFor="email">Email </label>
+                        <input value={email} type="email" placeholder="youremail@gmail.com" name="email" />
+                        <label htmlFor="password">Password </label>
+                        <input value={pass} type="password" placeholder="*****" name="password" />
+                        <button className='login-btn'>Log In</button>
+                    </form>
+                    <button className='register-btn' onClick={() => props.onFormSwitch('login')}>Login Here </button>
```
Comment (RealWeeks) [inline src/Register.jsx]: break this handler into it's own method

File: src/App.css [hunk 4]
```diff
@@ -1,38 +1,57 @@
 .App {
```
Comment (RealWeeks) [inline src/App.css:1]: in the future we want to bundle each component so it also has the css for it's parts inside

File: src/Register.jsx (hunk 2 above)
Comment (b-bold) [reply src/Register.jsx]: made this change in the code, i believe.

File: src/Login.jsx (hunk 1 above)
Comment (b-bold) [reply src/Login.jsx]: made this change in the code, i believe.

**Pull request merged**